src/server/pegasus_server_impl.cpp

```diff
@@ -822,7 +867,7 @@ void pegasus_server_impl::on_multi_get(const ::dsn::apps::multi_get_request &req
             it->SeekForPrev(stop);
             bool first_exclusive = !stop_inclusive;
             std::vector<::dsn::apps::key_value> reverse_kvs;
-            while (count < max_kv_count && size < max_kv_size && it->Valid()) {
+            while (iteration_count < max_iteration_count && size < max_kv_size && it->Valid()) {
```

Comment:
Should check `size < max_iteration_size` too ?

Reply:
Sure~ I miss to update condition here before